package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "react-canvas-knob",
-  "version": "0.4.0",
+  "name": "@oliviaboni/react-canvas-knob",
+  "version": "0.5.3",
   "description": "HTML5 Canvas-based React knob/dial component",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "start": "node_modules/webpack-dev-server/bin/webpack-dev-server.js --no-info",
-    "babel": "node_modules/babel-cli/lib/babel/index.js Knob.js -o index.js",
+    "start": "webpack-dev-server --no-info",
+    "build": "babel -d lib/ src/",
+    "prepublish": "npm run build",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": "git@github.com:joshjg/react-canvas-knob.git",
+  "repository": "git@github.com:oliviaboni/react-canvas-knob.git",
```

Comment:
Since you're going to do a merge request to @joshjg, we should leave this line as it is. 😅